tsl/src/compression/compression.c

```diff
@@ -1134,6 +1134,8 @@ create_per_compressed_column(TupleDesc in_desc, TupleDesc out_desc, Oid out_reli
 		/* find the mapping from compressed column to uncompressed column, setting
 		 * the index of columns that don't have an uncompressed version
 		 * (such as metadata) to -1
+		 * Assumption: column names are the same on compressed and
+		 *       uncompressed chunk.
```

Comment:
This is a pre-existing assumption. This code does not change any pre-existing assumptions.

Reply:
What happens if this assumption is not correct, does the code crash or does it just give an error?

Reply:
I added this comment to _clarify_ existing code and the assumptions made by it. This would crash if the rename were to go through without respecting the existing assumption.